joplin/static/js/admin.js

```diff
@@ -1,14 +1,18 @@
 document.addEventListener('DOMContentLoaded', function(event) {
   // HACK: I can't find a way to override this in python
-  document.querySelector('label[for=id_title]').textContent = 'Actionable Title';
+  const title = document.querySelector('label[for=id_title]');
+  if (title) {
+    title.textContent = 'Actionable Title';
+  }
```

Comment:
I couldn't find a better way to update the `title` help text because it's part of the base `Page` model. I'm sure there's something better, so I filed an issue cityofaustin/joplin#34